internal/tokens/team_token.go

```diff
@@ -0,0 +1,115 @@
+package tokens
+
+import (
+	"context"
+	"time"
+
+	"github.com/leg100/otf/internal"
+	"github.com/leg100/otf/internal/rbac"
+	"github.com/lestrrat-go/jwx/v2/jwk"
+)
+
+type (
+	// TeamToken provides information about an API token for a user.
+	TeamToken struct {
+		ID          string
+		CreatedAt   time.Time
+		Description string
+
+		// Token belongs to an team
+		Team string
```

Comment:
I suspect this should be something closer to `Team *team`, otherwise there's no easy way to look up the `Team` details from the Token, without manually doing extra DB calls.